Sources/StreamChatUI/CommonViews/CellSeperatorView/CellSeparatorCollectionViewLayout.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import UIKit
+
+/// A reusable collection view layout that supports rendering separator views in the collection cells.
+open class CellSeparatorCollectionViewLayout<ExtraData: ExtraDataTypes>: UICollectionViewFlowLayout {
+
+    /// The height of the cell separator view.
+    open var separatorHeight: CGFloat = 0.5
```

Comment:
nit: Probably should be `0.33`? 

<img width="257" alt="Snímek obrazovky 2021-03-02 v 0 22 24" src="https://user-images.githubusercontent.com/17381941/109572361-5e212380-7aed-11eb-8f85-1cb23f12ddf3.png">
 as default separator height is 0.33?

Reply:
I think the default actually is 1 / display scale